### ISSUE TITLE
Interoperability between different server implementations

### ISSUE THREAD

**Issue by LukasMasuch** [CovidShield/server#19]
I'm working on the German [Corona-Warn-App](https://github.com/corona-warn-app) project and currently investigating possibilities for interoperability between different exposure notification systems/backends. Do you already have plans on how the CovidShield server can exchange key data with other deployments? The [exposure notification reference server](https://github.com/google/exposure-notifications-server) provides a specification for a [federation API](https://github.com/google/exposure-notifications-server/blob/master/internal/pb/federation.proto) which can be used for an authorized exchange of daily keys (TEKs) between different server deployments. Is this federation API something you might consider to integrate into the CovidShield server?

**Comment by burke** [comment]
Yeah, we haven't implemented the federation API yet but I have an early WIP branch open for it somewhere—it definitely works with our data model and we intend to implement it (or accept PRs for it) when we need it.
